### DISCUSSION TITLE
Play multi animations

### DISCUSSION THREAD

**Discussion by beshoyfd** [google/model-viewer#4345] [Q&A]
hello , i am trying to play multi animations in parellal since package doesnt support that , i am trying to access instance of mixer from scene with the following code

`
  const $scene = Object.getOwnPropertySymbols(modelViewer)  .filter(symbol => symbol.description === 'scene')[0];
  const scene = modelViewer[$scene];
  console.log(scene);
  const clips = scene.animations;
  console.log(clips);
  console.log(scene.mixer);
  scene.mixer.clipAction(clips[0]).play();
`
i get instance of mixer and scene but animation doesn't play
![image](https://github.com/google/model-viewer/assets/5092392/89493d9a-d1f1-4594-a889-60dc856fdddf)

**Comment by elalish** [comment]
Sorry, we can't give support outside our public API. If you'd like to make a PR to add the feature you're interested in for all users, we'll be happy to help and review it.
